examples/hyperapp-kitchen-sink/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  rules: {
+    'no-unused-vars': [2, { varsIgnorePattern: 'h' }],
+    'react/jsx-uses-vars': 2,
+    'react/react-in-jsx-scope': 0,
+  },
+};
```

Comment:
Instead of this, you can add `/** @jsx Foo.bar */` pragma to files that use jsx
See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md

Reply:
We use this approach for mithril, and it works well for us

Reply:
that sounds like a good suggestion!